apps/hosts/views.py

```diff
@@ -189,3 +189,27 @@ def create_challenge_host_team(request):
         challenge_host.save()
         return Response(response_data, status=status.HTTP_201_CREATED)
     return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+@throttle_classes([UserRateThrottle, ])
+@api_view(['DELETE', ])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication, ))
+def delete_challenge_host_from_team_by_self(request, challenge_host_team_pk):
```

Comment:
can you please change the view name to `remove_self_from_challenge_host_team`. this actually signifies that the api is trying to remove a challenge host from challenge host team instead of deleting it.